Google.PowerShell/Compute/GceCmdlet.cs

```diff
@@ -110,6 +122,51 @@ private void WriteWarnings(Operation op)
             }
         }
 
+        private void WriteProgress(Operation op, string progressMessage)
+        {
+            int activityId = (int)(op.Id % int.MaxValue ?? 0);
+            string activity = progressMessage ?? op.Description;
+            if (activity == null)
+            {
+                string operationType = op.OperationType;
+                operationType = operationType.First().ToString().ToUpper() + operationType.Substring(1);
+                int startIndex = op.TargetLink.IndexOf("projects", StringComparison.Ordinal);
+                if (startIndex < 0 || startIndex >= op.TargetLink.Length)
+                {
+                    startIndex = 0;
+                }
+                activity = operationType + " " + op.TargetLink.Substring(startIndex);
+            }
+            string statusDescription = op.StatusMessage ?? op.Status;
+            int percentComplete;
+            if (op.Progress == null || op.Progress == 0)
+            {
+                percentComplete = _spinnerValue = (_spinnerValue + 1) % 100;
```

Comment:
That is assuming that Evil Jim, who will join the team later, reuses the same constant value. But Evil Jim won't do that. Instead, Evil Jim will have his cmdlets inherit from `EvilGceCmdlet` which have a sleep interval of 149ms. Why? Because Evil Jim is evil like that.

Since we won't be adding many more GCE cmdlets in the future, this is fine. But if there was a lot of future expansion, we'd need a stronger guarantee or a better solution for this.